### PR TITLE
[Do not review] Trying out path filters in pipeline

### LINKED ISSUE #36457: Don't run the CI if only .md files were changed
While reviewing #36431 where only `.md` were edited, I've noticed that the CI was triggered and all tests executed:

obraz

We had a similar problem in the performance repo in the past which was solved by: https://github.com/dotnet/performance/pull/355

I am an AzDO n00b and I could not find `azure-pipelines.yml` in this repo, so this is why I create an issue instead of sending a PR.

/cc @ViktorHofer

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/runtime/issues/36457

### CODE REVIEW

File: eng/pipelines/global-build.yml [hunk 1]
```diff
@@ -15,18 +15,14 @@ pr:
     - docs/manpages/*
     - eng/pipelines/global-build.yml
```
Comment (safern) [inline eng/pipelines/global-build.yml:16]: You need to exclude this one in order to test this as well.

File: eng/pipelines/runtime.yml [hunk 2]
```diff
@@ -37,13 +33,9 @@ pr:
...
-    - CODE-OF-CONDUCT.md
-    - CONTRIBUTING.md
-    - LICENSE.TXT
-    - PATENTS.TXT
-    - README.md
-    - SECURITY.md
-    - THIRD-PARTY-NOTICES.TXT
+    - /*.TXT
+    - /*.md
+    - eng/pipelines/runtime.yml
```
Comment (safern) [inline eng/pipelines/runtime.yml:38]: You need to exclude, all the .yml files you are touching to test this works.

**Comment by ViktorHofer** [comment]
It doesn't work :(

**Comment by safern** [comment]
> It doesn't work :(

Try changing it to:
```yml
- '*.md'
- '*.TXT'
```

I think that way you escape the '*' and it might work.

**Comment by safern** [comment]
> I think that way you escape the '*' and it might work.

Let's see, I just pushed that suggestion to see if it works.

**Comment by ViktorHofer** [comment]
@adamsitnik seems like we can't easily ignore all .MD files. I think there's a uservoice ticket somewhere against AzDO to support that. Closing.

**Comment by adamsitnik** [comment]
@ViktorHofer thanks for trying!

**Pull request closed without merging**